src/main/scala/mesosphere/marathon/Features.scala

```diff
@@ -11,10 +11,14 @@ object Features {
   //enable external volumes
   val EXTERNAL_VOLUMES = "external_volumes"
 
+  //enable secrets
+  val SECRETS = "secrets"
```

Comment:
should this (and the other feature name constants) also be declared as `lazy` to avoid init race conditions? seeing strange CI errors...